src/models/pokemon.js

```diff
@@ -17,9 +18,9 @@ class Pokemon extends Model {
     static DittoPokemonId = 132;
     static WeatherBoostMinLevel = 6;
     static WeatherBoostMinIvStat = 4;
-    static PokemonTimeUnseen = 1200;
-    static PokemonTimeReseen = 600;
-    static DittoDisguises = [46,163,165,167,187,223,293,316,322,399,590];
+    static PokemonTimeUnseen = config.dataparser.pokemonTimeUnseen * 60;
+    static PokemonTimeReseen = config.dataparser.pokemonTimeReseen * 60;
```

Comment:
Oh I realize this thing is not used yet. Somebody might need to look at RDM to see how to implement it.